apps/ledger-live-desktop/src/renderer/bridge/proxy.js

```diff
@@ -152,6 +152,10 @@ export const getAccountBridge = (
     prepareTransaction,
     sync,
     receive,
+    applyReconciliation: bridgeImpl.getAccountBridge(account, parentAccount).applyReconciliation,
+    toAccountRaw: bridgeImpl.getAccountBridge(account, parentAccount).toAccountRaw,
+    fromAccountRaw: bridgeImpl.getAccountBridge(account, parentAccount).fromAccountRaw,
+    mockAccount: bridgeImpl.getAccountBridge(account, parentAccount).mockAccount,
```

Comment:
share the result of `getAccountBridge` into one variable 🙏

Reply:
Fixed. Thanks for the feedback.